apiserver/allfacades.go

```diff
@@ -166,8 +166,9 @@ func AllFacades() *facade.Registry {
 	reg("LogForwarding", 1, logfwd.NewFacade)
 	reg("MachineActions", 1, machineactions.NewExternalFacade)
 
-	reg("MachineManager", 2, machinemanager.NewFacade)
-	reg("MachineManager", 3, machinemanager.NewFacade) // Version 3 adds DestroyMachine and ForceDestroyMachine.
+	reg("MachineManager", 2, machinemanager.NewFacadeV3)
```

Comment:
Someone got there first, I just merged a commit from develop doing what you've suggested.   Adding my changes to version 4 of the API.